Cargo.lock

```diff
@@ -1045,7 +1046,7 @@ checksum = "9a89dc7a5850d0e983be1ec2a463a171d20990487c3cfcd68b5363f1ee3d6fe0"
 dependencies = [
  "cfg-if",
  "hashbrown 0.14.3",
- "indexmap 2.2.1",
+ "indexmap 1.9.3",
```

Comment:
dependabot, stg you've got to get your act together (probably actually cargo that does this but whatever)